Crash when switching to Android Google TV home screen during playback

### Describe the bug

1. Start the TV
2. Go to the Jellyfin app, start playing a movie
3. Press the home button on your remote control
4. Once you are on the Android Google TV home screen, open the Jellyfin app again.

### Logs

***Stack Trace***: 
```log
java.lang.NullPointerException
	at org.jellyfin.androidtv.ui.home.HomeFragmentHelper.loadResume(HomeFragmentHelper.kt:33)
	at org.jellyfin.androidtv.ui.home.HomeFragmentHelper.loadResumeVideo(HomeFragmentHelper.kt:51)
	at org.jellyfin.androidtv.ui.home.HomeRowsFragment$onCreate$1.invokeSuspend(HomeRowsFragment.kt:123)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:103)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@2c64ccc, Dispatchers.IO]
```  
***Logcat***: 
```log
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 	Caused by: java.net.ConnectException: failed to connect to /192.168.178.32 (port 8096) from /192.168.178.33 (port 51112) after 6000ms: isConnected failed: EHOSTUNREACH (No route to host)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at libcore.io.IoBridge.isConnected(IoBridge.java:288)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at libcore.io.IoBridge.connectErrno(IoBridge.java:193)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at libcore.io.IoBridge.connect(IoBridge.java:135)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at java.net.PlainSocketImpl.socketConnect(PlainSocketImpl.java:142)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:390)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:230)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:212)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:436)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at java.net.Socket.connect(Socket.java:621)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at okhttp3.internal.platform.Platform.connectSocket(Platform.kt:128)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.kt:295)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		... 18 more
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 	Caused by: android.system.ErrnoException: isConnected failed: EHOSTUNREACH (No route to host)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		at libcore.io.IoBridge.isConnected(IoBridge.java:275)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 		... 28 more
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): Caused by: java.net.ConnectException: Failed to connect to /192.168.178.32:8096
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.kt:297)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.connection.RealConnection.connect(RealConnection.kt:207)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.kt:226)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.kt:106)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.kt:74)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.connection.RealCall.initExchange$okhttp(RealCall.kt:255)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.kt:32)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
11-11 09:27:43.926 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.kt:95)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.kt:83)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.kt:76)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:201)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:517)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at java.lang.Thread.run(Thread.java:919)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): Caused by: java.net.ConnectException: failed to connect to /192.168.178.32 (port 8096) from /192.168.178.33 (port 51110) after 6000ms: isConnected failed: EHOSTUNREACH (No route to host)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at libcore.io.IoBridge.isConnected(IoBridge.java:288)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at libcore.io.IoBridge.connectErrno(IoBridge.java:193)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at libcore.io.IoBridge.connect(IoBridge.java:135)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at java.net.PlainSocketImpl.socketConnect(PlainSocketImpl.java:142)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:390)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:230)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:212)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:436)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at java.net.Socket.connect(Socket.java:621)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.platform.Platform.connectSocket(Platform.kt:128)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.kt:295)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	... 18 more
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): Caused by: android.system.ErrnoException: isConnected failed: EHOSTUNREACH (No route to host)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	at libcore.io.IoBridge.isConnected(IoBridge.java:275)
11-11 09:27:43.927 W/LeanbackChannelWorker( 9065): 	... 28 more
11-11 09:27:43.939 I/WM-WorkerWrapper( 9065): Worker result RETRY for Work [ id=c1a31c48-009e-4162-94b6-37691ad72d4c, tags={ org.jellyfin.androidtv.integration.LeanbackChannelWorker } ]
11-11 09:27:47.635 W/lyfin.androidt( 9065): Suspending all threads took: 9.598ms
11-11 09:28:43.960 W/lyfin.androidt( 9065): No such thread id for suspend: 6
11-11 12:26:23.125 I/org.jellyfin.sdk.api.client.KtorClient( 9065): GET http://192.168.178.32:8096/Shows/NextUp?userId=94277486-19ff-4c2a-b614-97825bef378a&limit=10&fields=DateCreated&imageTypeLimit=1&enableTotalRecordCount=true&disableFirstEpisode=false&enableRewatching=false
11-11 12:26:23.135 I/org.jellyfin.sdk.api.client.KtorClient( 9065): GET http://192.168.178.32:8096/Users/94277486-19ff-4c2a-b614-97825bef378a/Items/Resume?limit=10&fields=DateCreated&mediaTypes=Video&imageTypeLimit=1&includeItemTypes=Episode&includeItemTypes=Movie&enableTotalRecordCount=true&enableImages=true&excludeActiveSessions=true
11-11 12:26:24.478 I/lyfin.androidt( 9065): Waiting for a blocking GC ProfileSaver
11-11 12:26:24.535 I/org.jellyfin.sdk.api.client.KtorClient( 9065): GET http://192.168.178.32:8096/Users/94277486-19ff-4c2a-b614-97825bef378a/Items/Latest?fields=Overview&includeItemTypes=Movie&isPlayed=false&limit=50&groupItems=true
11-11 12:26:24.542 I/org.jellyfin.sdk.api.client.KtorClient( 9065): GET http://192.168.178.32:8096/Users/94277486-19ff-4c2a-b614-97825bef378a/Items/Latest?fields=Overview&includeItemTypes=Episode&isPlayed=false&limit=50&groupItems=true
11-11 12:26:24.544 I/org.jellyfin.sdk.api.client.KtorClient( 9065): GET http://192.168.178.32:8096/Users/94277486-19ff-4c2a-b614-97825bef378a/Items/Latest?fields=Overview&includeItemTypes=Movie&includeItemTypes=Series&isPlayed=false&limit=50&groupItems=true
11-11 12:26:26.207 I/org.jellyfin.sdk.api.client.KtorClient( 9065): GET http://192.168.178.32:8096/Users/94277486-19ff-4c2a-b614-97825bef378a/Views?includeHidden=false
11-11 12:26:26.292 I/lyfin.androidt( 9065): WaitForGcToComplete blocked ProfileSaver on HeapTrim for 1.814s
11-11 12:26:27.278 I/MigrationContext( 9065): Requested migration from 7 to 7. Found 1 migrations in total.
11-11 12:26:27.279 D/LeanbackChannelWorker( 9065): Updating channel content://android.media.tv/channel/209
11-11 12:26:27.395 D/LeanbackChannelWorker( 9065): Updating channel content://android.media.tv/channel/207
11-11 12:26:27.873 D/LeanbackChannelWorker( 9065): Updating channel content://android.media.tv/channel/210
11-11 12:26:28.377 D/LeanbackChannelWorker( 9065): Updating channel content://android.media.tv/channel/211
11-11 12:26:28.423 D/LeanbackChannelWorker( 9065): Updating channel content://android.media.tv/channel/208
11-11 12:26:28.534 I/WM-WorkerWrapper( 9065): Worker result SUCCESS for Work [ id=c1a31c48-009e-4162-94b6-37691ad72d4c, tags={ org.jellyfin.androidtv.integration.LeanbackChannelWorker } ]
11-11 12:26:30.660 W/ActivityThread( 9065): handleWindowVisibility: no activity for token android.os.BinderProxy@2749772
11-11 12:26:30.846 I/ActivityThemeExtensionsKt( 9065): Applying theme DARK
11-11 12:26:30.880 D/ACRA    ( 9065): onActivityCreated class org.jellyfin.androidtv.ui.browsing.MainActivity
11-11 12:26:30.915 D/NavigationRepositoryImpl( 9065): Navigating to Fragment(fragment=class org.jellyfin.androidtv.ui.home.HomeFragment (Kotlin reflection is not available), arguments=Bundle[{}]) (via init)
11-11 12:26:30.954 D/LegacyMediaManager( 9065): Added event listener.  Total listeners: 1
11-11 12:26:30.957 W/MainActivity( 9065): Activity org.jellyfin.androidtv.ui.browsing.MainActivity started without a session, bouncing to StartupActivity
11-11 12:26:30.970 E/ACRA    ( 9065): ACRA caught a NullPointerException for org.jellyfin.androidtv
11-11 12:26:30.970 E/ACRA    ( 9065): java.lang.NullPointerException
11-11 12:26:30.970 E/ACRA    ( 9065): 	at org.jellyfin.androidtv.ui.home.HomeFragmentHelper.loadResume(HomeFragmentHelper.kt:33)
11-11 12:26:30.970 E/ACRA    ( 9065): 	at org.jellyfin.androidtv.ui.home.HomeFragmentHelper.loadResumeVideo(HomeFragmentHelper.kt:51)
11-11 12:26:30.970 E/ACRA    ( 9065): 	at org.jellyfin.androidtv.ui.home.HomeRowsFragment$onCreate$1.invokeSuspend(HomeRowsFragment.kt:123)
11-11 12:26:30.970 E/ACRA    ( 9065): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
11-11 12:26:30.970 E/ACRA    ( 9065): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
11-11 12:26:30.970 E/ACRA    ( 9065): 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
11-11 12:26:30.970 E/ACRA    ( 9065): 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:103)
11-11 12:26:30.970 E/ACRA    ( 9065): 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
11-11 12:26:30.970 E/ACRA    ( 9065): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
11-11 12:26:30.970 E/ACRA    ( 9065): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
11-11 12:26:30.970 E/ACRA    ( 9065): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
11-11 12:26:30.970 E/ACRA    ( 9065): 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@2c64ccc, Dispatchers.IO]
11-11 12:26:30.970 D/ACRA    ( 9065): Building report
11-11 12:26:30.973 D/ACRA    ( 9065): Starting collectors with priority FIRST
11-11 12:26:30.976 D/ACRA    ( 9065): Calling collector org.acra.collector.StacktraceCollector
11-11 12:26:30.977 D/ACRA    ( 9065): Calling collector org.acra.collector.DropBoxCollector
11-11 12:26:30.978 D/ACRA    ( 9065): Collector org.acra.collector.DropBoxCollector completed
11-11 12:26:30.978 D/ACRA    ( 9065): Calling collector org.acra.collector.LogCatCollector
11-11 12:26:30.979 D/ACRA    ( 9065): Collector org.acra.collector.StacktraceCollector completed
11-11 12:26:30.987 D/ACRA    ( 9065): Retrieving logcat output (buffer:default)...
11-11 12:26:31.028 E/libc    (13625): Access denied finding property "persist.vendor.system.svp.dbg.loglevel"
```  

### App information

***App version***: `0.16.0-beta.5` (`160005`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"mt5895","BOOTLOADER":"unknown","BRAND":"Sony","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"BRAVIA_VH2","DISPLAY":"BRAVIA_VH2_M_EU-user 10 QTG3.200305.006.S346 672821 release-keys","FINGERPRINT":"Sony\/BRAVIA_VH2_M_EU\/BRAVIA_VH2:10\/QTG3.200305.006.S346\/672821:user\/release-keys","HARDWARE":"mt5895","HOST":"BuildHost731","ID":"QTG3.200305.006.S346","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"Sony","MODEL":"BRAVIA 4K VH2","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"BRAVIA_VH2_M_EU","RADIO":"unknown","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1690485385000,"TYPE":"user","UNKNOWN":"unknown","USER":"root","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","FIRST_SDK_INT":29,"INCREMENTAL":"672821","PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"10","RESOURCES_SDK_INT":29,"SDK":"29","SDK_INT":29,"SECURITY_PATCH":"2023-06-01"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":160005,"VERSION_NAME":"0.16.0-beta.5"}
```  

### Device information

***Android version***: `10`  
***Device brand***: `Sony`  
***Device product***: `BRAVIA_VH2_M_EU`  
***Device model***: `BRAVIA 4K VH2`  

### Crash information

***Start time***: `2023-11-11T02:33:56.300+01:00`  
***Crash time***: `2023-11-11T12:26:31.209+01:00`  

### Application version

0.16.0-beta.5

### Where did you install the app from?

Google Play

### Device information

Sony XR-83A90J

### Android version

Android 10

### Jellyfin server version

10.8.12

--- a/app/src/main/java/org/jellyfin/androidtv/ui/home/HomeRowsFragment.kt
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/home/HomeRowsFragment.kt
@@ -17,11 +17,14 @@ import androidx.lifecycle.repeatOnLifecycle
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.awaitCancellation
 import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.filterNotNull
+import kotlinx.coroutines.flow.first
 import kotlinx.coroutines.flow.launchIn
 import kotlinx.coroutines.flow.onEach
 import kotlinx.coroutines.isActive
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
+import kotlinx.coroutines.withTimeout
 import org.jellyfin.androidtv.auth.repository.UserRepository
 import org.jellyfin.androidtv.constant.CustomMessage
 import org.jellyfin.androidtv.constant.HomeSectionType
@@ -88,15 +91,17 @@ class HomeRowsFragment : RowsSupportFragment(), AudioEventListener, View.OnKeyLi
 
 		adapter = MutableObjectAdapter<Row>(PositionableListRowPresenter())
 
-		val currentUser = userRepository.currentUser.value
-
 		lifecycleScope.launch(Dispatchers.IO) {
+			val currentUser = withTimeout(30.seconds) {
+				userRepository.currentUser.filterNotNull().first()
+			}
+
 			// Start out with default sections
 			val homesections = userSettingPreferences.homesections
 			var includeLiveTvRows = false
 
 			// Check for live TV support
-			if (homesections.contains(HomeSectionType.LIVE_TV) && currentUser?.policy?.enableLiveTvAccess == true) {
+			if (homesections.contains(HomeSectionType.LIVE_TV) && currentUser.policy?.enableLiveTvAccess == true) {
 				// This is kind of ugly, but it mirrors how web handles the live TV rows on the home screen
 				// If we can retrieve one live TV recommendation, then we should display the rows
 				val recommendedPrograms by api.liveTvApi.getRecommendedPrograms(

